source/agora/utils/Utility.d

```diff
@@ -227,3 +227,47 @@ public struct UbyteHexString
         });
     }
 }
+
+/// Utility class to implement vibe.d ResultSerializer.serialize and
+/// ResultSerializer.serialize templates
+public class ResultSerializerUtil
+{
+    /***************************************************************************
+
+        Serialize the string as it is without any modifications
+        like JSON escaping
+
+        Params:
+            Policy: serialization policy type
+            InputT: input data type (string)
+            OutputRangeT: output range type
+            output_range: output range to serialize the string into
+            input_string: string to serialize
+
+    ***************************************************************************/
+
+    public static void raw_string_serialize(alias Policy, InputT, OutputRangeT)(OutputRangeT output_range, InputT input_string)
+    {
+        import std.string;
+        output_range ~= input_string.representation;
+    }
+
+    /***************************************************************************
+
+        Deserialize the string that was previouly serialized by
+        raw_string_serialize
+
+        Params:
+            Policy: serialization policy type
+            ReturnT: return type (string)
+            InputRangeT: input range type
+            input_range: input range
+
+    ***************************************************************************/
+
+    public static ReturnT raw_string_deserialize(alias Policy, ReturnT, InputRangeT)(InputRangeT input_range)
+    {
+        import std.string;
+        return std.string.assumeUTF(input_range);
```

Comment:
I think you need to validate here otherwise an exception might be thrown down the line, no ?